### ISSUE TITLE
Training settings in the paper differ from the code

### ISSUE THREAD

**Issue by aapanaetov** [wzhouxiff/RestoreFormer#15]
Hi! Thank you for publishing such an amazing work! In the paper you set learning rate for HQ dictionary to 7e-5 while in the HQ_Dictionary.yaml learning rate is set to 4.5e-6.  Schedule steps, learning rate and loss weights in the configs for both HQ dictionary and RestoreFormer are different from the paper. Which settings should I use to reproduce your excellent results?

**Comment by wzhouxiff** [comment]
Please follow the setting described in the paper. Note that the learning rate set in the config is not the actual learning rate. It will be divided by the number of gpus used. The learning rate described in the paper is the one after dividing.
